### ISSUE TITLE
Optional CLI flags result in interactive prompt

### ISSUE THREAD

**Issue by Dilski** [Noovolari/leapp#394]
**Is your feature request related to a problem? Please describe.**
When using the `leapp session start-ssm-session` command, the region, session id and instance id flags are "all or nothing" (i.e you have to provide all the flags, or none of the flags and select values interactively).

In my use case, the region and session id remain the same, but I only know the name of my instance and not it's id. 

**Describe the solution you'd like**
I would like to either/and:
* Select _just_ the instance interactively (i.e missing a flag prompts interaction)
* Pass my instance name as a flag

**Additional context**
https://github.com/Noovolari/leapp/blob/9333ea48103e01f1b80583397c0e8bbec7bd6713/packages/cli/src/commands/session/start-ssm-session.ts#L119

**Comment by andreacavagna01** [comment]
Good point! Thanks for pointing that out! Yeah, this is the feature flag to change! You can add your own PR or we will start working on it in some weeks 👍
